### PR TITLE
add Block Pore Volumes

### PR DESCRIPTION
**Pull request by goncalvesmachadoc**

### CODE REVIEW

File: src/opm/input/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp [hunk 1]
```diff
@@ -141,15 +141,15 @@ struct SummaryConfigContext {
...
          {"OIPL" , {"ROIPL" ,"FOIPL" }},
          {"OIPG" , {"ROIPG" ,"FOIPG"}},
          {"GIP"  , {"RGIP" , "FGIP"}},
-         {"GIPR" , {"FGIPR"}},
+         {"GIPR" , {"FGIPR", "BGPV"}},
          {"GIPL" , {"RGIPL" , "FGIPL"}},
          {"GIPG" , {"RGIPG", "FGIPG"}},
          {"WIP"  , {"RWIP" , "FWIP"}},
-         {"WIPR" , {"FWIPR"}},
+         {"WIPR" , {"FWIPR", "BWPV"}},
```
Comment (bska) [inline src/opm/input/eclipse/EclipseState/SummaryConfig/SummaryConfig.cpp:152]: I don't think you really need this part of the change.  The `required_fields` track which solution variables must be available in order to calculate certain derived summary quantities.  For instance, the `PRESSURE` solution variable must exist in order to calculate the average field pressure (`FPR`), the average region pressure values (`RPR`) or extract pressure values at the block level (`BPR`).

As your OPM-simulators PR (OPM/opm-simulators#3905) proves, you can compute the `BxPV` quantities without any of the `xIPR` quantities in place.  That said, adding these values here doesn't hurt anything but it also doesn't help.

**Review by bska — COMMENTED** [review]
This looks good to me for the most part.  I believe you get the same result if you don't include the changes to `SummaryConfig.cpp`, but I'm not going to insist that that part be removed either.

**Comment by bska** [comment]
jenkins build this opm-simulators=3905 please

**Comment by goncalvesmachadoc** [comment]
jenkins build this please

**Comment by bska** [comment]
Thanks for the updates @goncalvesmachadoc!  This looks good to me now and I'll merge this into master now.

**Pull request merged**